### ISSUE TITLE
Rules appear to save after hitting "cancel" instead of "save rule" for tanks and traps

### ISSUE THREAD

**Issue by CaseyDengler** [patrickomni/omnimobileserver#29]
Steps to Reproduce:
1) Log into Omni
2) Impersonate tanks or traps
3) Open the rules box
4) Make a change to the rules
5) Click on "cancel"
6) Open the rules up again

Expected:
The rules should appear as they were before any changes were made, as leaving the page like that give the illusion that changes did occur, whether or not they did.

Occurred:
The rules that were changed but not saved are still shown as being changed.

**Comment by christopherdude** [comment]
This was a great find.  This seems to now (as of tonight) only happen on those WITHOUT rules already set.  Fixed.. also, since there's an alert that shows, the background before wasn't "dimming" or "darkening" out.  I found myself trying to click on items, but couldn't since I had to click on the OK on the pseudo msgbox.  I made the background behind the msgbox go dark a little (see attached where the star is) to be more obvious.
![capture](https://cloud.githubusercontent.com/assets/9539359/7698053/f965dbc4-fdc1-11e4-81d1-7e75b894a6f2.PNG)
